src/datepicker/datepicker.ts

```diff
@@ -33,7 +34,7 @@ import {NgbDatepickerI18n} from './datepicker-i18n';
 import {isChangedDate, isChangedMonth} from './datepicker-tools';
 import {hasClassName} from '../util/util';
 
-const NGB_DATEPICKER_VALUE_ACCESSOR = {
+export const NGB_DATEPICKER_VALUE_ACCESSOR = {
```

Comment:
why `export` ?

Reply:
It is required in datepicker-month-view.spec.ts:228